pkg/database/db.go

```
@@ -45,6 +45,12 @@ func Init() {
 	if !DB.Migrator().HasTable(&models.FavoriteQuickstart{}) {
 		DB.Migrator().CreateTable(&models.FavoriteQuickstart{})
 	}
+	if (!DB.Migrator().HasTable(&models.QuickstartTag{})) {
+		DB.Migrator().CreateTable(&models.QuickstartTag{})
+	}
+
+	DB.SetupJoinTable(&models.Quickstart{}, "Tags", &models.QuickstartTag{})
```

golangci-lint (2.12.2)

[error] Error return value of `DB.SetupJoinTable` is not checked

(errcheck)